### PR TITLE
Editorial rewrite of section on charter approval

### PR DESCRIPTION
**Pull request by frivoal**
The original text was messy and hard to follow, as well as hard to build upon. This commit does not make any substantive change, but improves the readability of this section.

***
<a href="https://pr-preview.s3.amazonaws.com/frivoal/w3process/pull/850.html" title="Last updated on Apr 23, 2024, 2:11 AM UTC (7e4b494)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/850/bb1c8c1...frivoal:7e4b494.html" title="Last updated on Apr 23, 2024, 2:11 AM UTC (7e4b494)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1861,39 +1861,44 @@ Content of a Charter</h3>
...
+	<em class=rfc2119>may</em> be approved by a [=Team Decision=].
+	The following are examples of substantive changes that would not require an [=Advisory Committee Review=]:
+	the addition of an in-scope deliverable,
+	a change of [=Team Contact=],
+	or a change of [=Chair=].
+
+	Such changes must be announced
+	to the [=Advisory Committee=]
+	and to <a href="#wgparticipant">participants in the Working</a> or <a href="#igparticipant">in the Interest Group</a>,
+	and a rationale must be provided.
```
Comment (TallTed) [inline index.bs:1901]:
```suggestion
	<em class=rfc2119>may</em> be approved by a [=Team Decision=]
	and would not require an [=Advisory Committee Review=].
	The following are examples of such changes:
	the addition of an in-scope deliverable,
	a change of [=Team Contact=],
	or a change of [=Chair=].

	Though [=Advisory Committee Review=] is not required,
	such changes must still be announced
	to the [=Advisory Committee=],
	and to participants in the <a href="#wgparticipant">Working</a> or <a href="#igparticipant">Interest</a> Group,
	and a rationale must be provided.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
massaging language for clarity ... and adding missing space

File: index.bs [hunk 2]
```diff
@@ -1861,39 +1861,44 @@ Content of a Charter</h3>
...
-	a change of [=Team Contact=],
-	or a change of [=Chair=].
-	Such changes must nonetheless be announced
-	to the [=Advisory Committee=]
-	and to <a href="#wgparticipant"> participants in the Working</a> or <a href="#igparticipant">in the Interest Group</a>,
-	and a rationale must be provided.
+	Any new [=charter=] (including re-chartering of existing [=chartered groups=])
+	and most substantive changes to an already-approved [=charter=]
+	of a [=Working Group=] or [=Interest Group=]
+	<em class=rfc2119>must</em> be approved by a [=W3C Decision=] following an [=AC Review=] of that [=charter=].
```
[the remark this answers is not in the document]
Comment (frivoal) [reply index.bs:1872]: I tend to prefer whichever markup pattern is shortest without those that are valid HTML. HTML allows this pattern, so that's why I go for it when possible.
https://html.spec.whatwg.org/multipage/syntax.html#unquoted

For this particular markup pattern, I don't expect the class attribute to be expanded with additional classes, so the `"` wouldn't even help preparing for their addition, so I'm happy to save a little bit of line noise, even if it's minor.

With that said, I wouldn't object using quotes here if others have a preference the other way around, it's only a minor preference of mine. And as you say, they get added when the output is generated anyway, so that makes it matter even less.

File: index.bs [hunk 3]
```diff
@@ -1861,39 +1861,46 @@ Content of a Charter</h3>
...
+	that do not affect the way the group functions in any significant way
+	<em class=rfc2119>may</em> be approved by a [=Team Decision=]
+	and would not require an [=Advisory Committee Review=].
+	The following are examples of such changes:
+	the addition of an in-scope deliverable,
+	a change of [=Team Contact=],
+	or a change of [=Chair=].
+
+	Though [=Advisory Committee Review=] is not required,
+	such changes must still be announced
```
Comment (frivoal) [inline index.bs:1900]:
```suggestion
	such changes <em class=rfc2119>must</em> still be announced
```

File: index.bs (hunk 2 above)
[the remark this answers is not in the document]
Comment (frivoal) [reply index.bs:1872]: @fantasai, as co-editor, agrees with the above logic, so closing this comment. @TallTed, if you really care, you could open a separate issue. My expectation is that we'll get consistent eventually (since both editors agree on the prefered approach), but I don't think it's worth making a giant edit to make it consistent in one go.

File: index.bs [hunk 4]
```diff
@@ -1861,39 +1861,46 @@ Content of a Charter</h3>
...
-	in response to the Call for Review.
-	Upon receipt of any such request,
-	the [=Team=] <em class="rfc2119">must</em> ensure
-	that the Call for Participation for the Working Group
-	occurs at least 60 days
-	after the Call for Review of the charter.
+	Substantive changes to a [=charter=] (including [[#charter-extension|extensions]])
+	that do not affect the way the group functions in any significant way
+	<em class=rfc2119>may</em> be approved by a [=Team Decision=]
+	and would not require an [=Advisory Committee Review=].
```
Comment (fantasai) [inline index.bs:1893]: This creates the sentence "Substantive changes to a [=charter=] would not require an [=AC Review=]" which is weird and not quite what we want to say.

Comment (fantasai) [inline index.bs:1893]:
```suggestion
	Substantive changes to a [=charter=] (including [[#charter-extension|extensions]])
	that do not affect the way the group functions in any significant way
	<em class=rfc2119>may</em> be approved by a [=Team Decision=],
	in which case they do not require an [=Advisory Committee Review=].
```
